WooCommerce/UITestsFoundation/Screens/Orders/UnifiedOrderScreen.swift

```diff
@@ -75,6 +83,14 @@ public final class UnifiedOrderScreen: ScreenObject {
     ///
     private var addNoteButton: XCUIElement { addNoteButtonGetter(app) }
 
+    /// Fixed Amount in Custom Amount ssheet.
```

Comment:
Typo here `ssheet`

Reply:
Nice catch! 🙇 
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/11274/commits/00304c897315a0662deeccf09306301fd414bf47.